package.json

```diff
@@ -20,6 +20,7 @@
   },
   "type": "module",
   "dependencies": {
+    "config": "^3.3.7",
```

Comment:
Why is this needed?

Reply:
It must of put it in when I ran the config build.

I'll edit it out.